### PR TITLE
Sum further than 2 numbers

### PR DESCRIPTION
**Pull request by robsidian**
Added a new function to solve issue #6

### CODE REVIEW

**Comment by robsidian** [comment]
Could you take a look @zrxdaly

File: model.py [hunk 1]
```diff
@@ -3,3 +3,5 @@
 def sum_function(sum1,sum2):
     return sum1 + sum2
 
+def sum_further(sum1,sum2,sum3):
```
Comment (zrxdaly) [inline model.py:6]: could you add a doctrine of this function

**Review by zrxdaly — APPROVED** [review]
this looks good

**Comment by zrxdaly** [comment]
Thanks for contribution @robsidian

**Review by zrxdaly — APPROVED** [review]

**Pull request merged**